pkg/services/notary/notary.go

```diff
@@ -345,21 +345,6 @@ func (n *Notary) OnNewRequest(payload *payload.P2PNotaryRequest) {
 	}
 }
 
-// safeCopy creates a copy of provided transaction by dereferencing it and creating
-// fresh witnesses so that the tx's witnesses may be modified without affecting the
-// copy's ones.
-func safeCopy(tx *transaction.Transaction) *transaction.Transaction {
```

Comment:
IIRC, it's not the only place where `Copy()` should be used.

Reply:
checked all `copy` places with this word. Do you have any other suggestions to find its usage?

Reply:
I thought about Oracle service, but it doesn't need this new `Copy()`, so OK for now.